homework-1/js/script.js

```diff
@@ -0,0 +1,136 @@
+////////////menu///////////
+const iconMenu = document.querySelector('.menu__icon');
+  if(iconMenu) {
+    const menuBody = document.querySelector('.menu__body');
+    iconMenu.addEventListener("click", function(e){
+      document.body.classList.toggle('lock');
+      iconMenu.classList.toggle('active');
+      menuBody.classList.toggle('active');
+    });
+  }
+
+  window.addEventListener('scroll', () => {
+    let scrollDistance = window.scrollY;
+      document.querySelectorAll('.section').forEach((el, i) => {
+        if (el.offsetTop - document.querySelector('#banner-submenu').clientHeight <= scrollDistance) {
+          document.querySelectorAll('.submenu a').forEach((el) => {
+            if (el.classList.contains('tips-active')) {
+              el.classList.remove('tips-active');
+            }
+          });
+  
+          document.querySelectorAll('.submenu li')[i].querySelector('a').classList.add('tips-active');
+        }
+      });
+  });
+///////////accordeon///////////////
+
+const tipLink = document.querySelectorAll('.tips__link');
+const tipSub = document.querySelectorAll('.block-text');
+const tipsItemWrap = document.querySelectorAll('.tips-item-wrap');
+
+for (let i = 0; i < tipLink.length; i++) {
+  tipLink[i].addEventListener("click", () => {
+    tipSub[i].classList.toggle('hidden');
+    tipsItemWrap[i].classList.toggle('hidden');
+    console.log(tipsItemWrap[i]);
+    tipLink[i].classList.toggle('tips__link_active');
+    
+  })
+}
+
+const fqasLink = document.querySelectorAll('.fqas__link');
+const tipSubElem = document.querySelectorAll('.block-text-next');
+
+for (let i = 0; i < fqasLink.length; i++) {
+  fqasLink[i].addEventListener("click", () => {
+    tipSubElem[i].classList.toggle('hidden');
+    fqasLink[i].classList.toggle('fqas__link_active');
+   })
+}
+////////tabs scroll
+
+window.addEventListener("scroll", function() {
+  var headerH = document.getElementById("banner").clientHeight - document.getElementById("header").clientHeight;
```

Comment:
What mean 'H' in the end variable name? Please don't be afraid to use full words for variables names , it is easier to understend what happening in code.
And also try to avoid 'var', use  'let' or 'const',
Don't forget about quotes

Reply:
fixed a bug

Reply:
fixed a bug

Reply:
done